src/java.base/unix/native/libnio/fs/UnixNativeDispatcher.c

```diff
@@ -687,11 +687,11 @@ Java_sun_nio_fs_UnixNativeDispatcher_lstat0(JNIEnv* env, jclass this,
         RESTARTABLE(statx_wrapper(AT_FDCWD, path, flags, mask, &statx_buf), err);
         if (err == 0) {
             copy_statx_attributes(env, &statx_buf, attrs);
-        } else {
+            return;
+        } else if (errno != ENOSYS && errno != EPERM) {
```

Comment:
Why is `ENOSYS` added here? Has this also been observed?

Reply:
No, I do not observe `ENOSYS`, but it is a common error code for an unsupported syscall. It is similar to copy_file_range.